src/engines/kotor/gui/gui.cpp

```diff
@@ -171,7 +171,12 @@ void GUI::createWidget(WidgetContext &ctx) {
 	else
 		throw Common::Exception("No such widget type %d", ctx.type);
 
-	ctx.widget->load(*ctx.strct);
+	try {
+		ctx.widget->load(*ctx.strct);
+	}
+	catch (Common::Exception &ex) {
+		Common::printException(ex, "WARNING: ");
+	}
```

Comment:
The `catch` needs to go on the same line as the closing brace. Also, can you use `Common::exceptionDispatcherWarning()` instead?

I.e.

```
try {
	ctx.widget->load(*ctx.strct);
} catch (...) {
	Common::exceptionDispatcherWarning();
}
```